tests/dialers_test.go

```diff
@@ -59,3 +83,38 @@ func TestDialer(t *testing.T) {
 	// there was no error returned.
 	db.Close()
 }
+
+func clientFromCredentials() (*http.Client, error) {
+
+	const SQLScope = "https://www.googleapis.com/auth/sqlservice.admin"
+
+	ctx := context.Background()
+
+	var client *http.Client
+
+	if f := *tokenFile; f != "" {
+
+		all, err := ioutil.ReadFile(f)
+		if err != nil {
+			return nil, fmt.Errorf("invalid json file %q: %v", f, err)
+		}
+		cfg, err := goauth.JWTConfigFromJSON(all, SQLScope)
+		if err != nil {
+			return nil, fmt.Errorf("invalid json file %q: %v", f, err)
+		}
+		t, _ := cfg.TokenSource(ctx).Token()
```

Comment:
Instead of pulling out the oauth2 package explicitly in this branch, how about using the Client method on the JWT config?

    client = cfg.Client(ctx)